### PR TITLE
Math Magicians: refactor with hooks

### PR DESCRIPTION
**Pull request by Lucas-Erkana**
- No linter errors ✔️
- Use correct Gitflow ✔️
- Refactor the (components/Calculator.js) from a class-based component to a functional component with hooks. ✔️ 
- After the refactor is done, test the app in the browser. It should keep the same functionality. ✔️

### CODE REVIEW

File: src/components/eachBtn.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
+import React from 'react';
```
Comment (Gambit142) [inline src/components/eachBtn.js]: - [x] Kindly remove this `Eslint` disable comment from this file. `Eslint` helps us to write clean, maintainable and readable codes and using `Eslint` disabling statements defeats this purpose. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Lucash2022 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Refactored components to functional-based components ✔️ 
- Use of React hooks ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/eachBtn.js (hunk 1 above)
Comment (Lucas-Erkana) [reply src/components/eachBtn.js]: @Gambit142 thank you so much for your review, I have enabled Eslint with this [commit](https://github.com/Lucash2022/math_magician_react/pull/4/commits/3367062dac583e4ecebff0f5dce9087de5d71aba)

**Review by Munsa1 — APPROVED** [review]
Hi @Lucash2022 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**